web-console/src/lib/functions/felderaRelation.ts

```diff
@@ -0,0 +1,28 @@
+import { Field, Relation } from '$lib/services/manager'
+import invariant from 'tiny-invariant'
+
+export type CaseDependentName = {
+  case_sensitive?: boolean | undefined
```

Comment:
why does this have to support "undefined"?

Reply:
Actually, in this case field can omit explicit 'undefined' while still being optional via `?:`.
This is in order to match a subset of type definitions of auto-generated types in `web-console/src/lib/services/manager/models/Relation.ts` and `web-console/src/lib/services/manager/models/Field.ts` where case_sensitive is optional